packages/core/src/logger.ts

```diff
@@ -1,6 +1,6 @@
 import loglevel from 'loglevel';
 
 const log = loglevel.getLogger('lk-components-js');
-log.setDefaultLevel('DEBUG');
+log.setDefaultLevel('ERROR');
```

Comment:
how about `warn` ?

Reply:
Now I have to change the branch name and the PR title :D
Warning is fine by me 👍